### DISCUSSION TITLE
[Feature] Provide way to filter for photos with unnamed people

### DISCUSSION THREAD

**Discussion by pedropombeiro** [immich-app/immich#5801] [Feature Request]
### The feature

It would be useful to be able to see only photos that are needing categorization (i.e. that have people that haven't yet been named). Maybe we could even sort from the smallest number of people in the photo, to the largest number of people in the photo (which is normally a photo in a crowd that you don't want to categorize anyway). This could be exposed for instance in the Explore section.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10534
